packages/tldraw/src/lib/canvas/TldrawSelectionForeground.tsx

```diff
@@ -35,6 +38,22 @@ export const TldrawSelectionForeground = track(function TldrawSelectionForegroun
 	const bottomRightEvents = useSelectionEvents('bottom_right')
 	const bottomLeftEvents = useSelectionEvents('bottom_left')
 
+	useEffect(() => {
+		function handleEvent(info: TLEventInfo) {
+			if (info.type === 'keyboard') {
+				if (info.ctrlKey !== isCtrlKeyPressed) {
+					setIsCtrlKeyPressed(info.ctrlKey)
+				}
+			}
+		}
+		editor.addListener('event', handleEvent)
+		return () => {
+			editor.removeListener('event', handleEvent)
+		}
+	}, [editor, isCtrlKeyPressed])
+
+	const canEnterCropMode = isCtrlKeyPressed && getShouldEnterCropMode(editor)
+
```

Comment:
We should really make editor.inputs into an atom, huh

Reply:
Just took a shot at it, that would be a big old change, let's land this and then think about that.